README.md

```diff
@@ -1 +1,70 @@
-# webpack
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+ToDo App
+> A todo webapp configured with webpack
+
+
+## Built With
+
+- Major languages
+  -Java Script 
+HTML 
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+   Clone the website in yoiur repost
+
+### Install
+  rum np
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/betses)
+- Twitter: [@twitterhandle](https://twitter.com/betse_s)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- Please, kindly update this issue page to to refer to the To Do list repo's issues page.